src/ralph/ui/forms/addresses.py

```diff
@@ -388,6 +389,10 @@ def __init__(self, *args, **kwargs):
     widget = IPWithHostWidget()
 
     def compress(self, value):
+        if value:
+            hostname, ip_number = value
+            if not ip_number:
+                raise ValidationError(_('IP Address is required'))
```

Comment:
The ``compress`` is a standard method on this field. It is defined by django and need not to be documented by me.